### PR TITLE
i18n(es): Update `configuration`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Updated `configuration`

### CODE REVIEW

File: docs/src/content/docs/es/reference/configuration.mdx [hunk 1]
```diff
@@ -346,6 +345,72 @@ starlight({
 });
 ```
 
+### `expressiveCode`
+
+**tipo:** `StarlightExpressiveCodeOptions | boolean`  
+**por defecto:** `true`
+
+Starlight usa [Expressive Code](https://github.com/expressive-code/expressive-code/tree/main/packages/astro-expressive-code) para renderizar bloques de código y agregar soporte para resaltar partes de ejemplos de código, agregar nombres de archivo a bloques de código y más.
+Vee la [guía de “Bloques de código”](/es/guides/authoring-content/#bloques-de-código) para aprender a usar la sintaxis de Expressive Code en tu contenido Markdown y MDX.
```
Comment (Waxer59) [inline docs/src/content/docs/es/reference/configuration.mdx:354]:
```suggestion
Consulta la [guía de “Bloques de código”](/es/guides/authoring-content/#bloques-de-código) para aprender a usar la sintaxis de Expressive Code en tu contenido Markdown y MDX.
```

File: docs/src/content/docs/es/reference/configuration.mdx [hunk 2]
```diff
@@ -346,6 +345,72 @@ starlight({
...
+
+Adicionalmente a las opciones estándar de Expressive Code, también puedes establecer las siguientes propiedades específicas de Starlight en tu configuración `expressiveCode` para personalizar aún más el comportamiento del tema para tus bloques de código:
+
+#### `themes`
+
+**tipo:** `Array<string | ThemeObject | ExpressiveCodeTheme>`  
+**por defecto:** `['starlight-dark', 'starlight-light']`
+
+Establece los temas utilizados para dar estilo a los bloques de código.
+Vee la [documentación de temas de Expressive Code](https://github.com/expressive-code/expressive-code/blob/main/packages/astro-expressive-code/README.md#themes) para obtener detalles de los formatos de tema admitidos.
```
Comment (Waxer59) [inline docs/src/content/docs/es/reference/configuration.mdx:383]:
```suggestion
Consulta la [documentación de temas de Expressive Code](https://github.com/expressive-code/expressive-code/blob/main/packages/astro-expressive-code/README.md#themes) para obtener detalles de los formatos de tema admitidos.
```

File: docs/src/content/docs/es/reference/configuration.mdx [hunk 3]
```diff
@@ -346,6 +345,72 @@ starlight({
...
+
+Starlight usa las variantes oscura y clara del [tema Night Owl](https://github.com/sdras/night-owl-vscode-theme) de Sarah Drasner por defecto.
```
Comment (Waxer59) [inline docs/src/content/docs/es/reference/configuration.mdx:385]:
```suggestion
Starlight usa por defecto las variantes oscura y clara del [tema Night Owl](https://github.com/sdras/night-owl-vscode-theme) de Sarah Drasner.
```

File: docs/src/content/docs/es/reference/configuration.mdx [hunk 4]
```diff
@@ -346,6 +345,72 @@ starlight({
...
+:::note
+Cuando estableces `themes`, debes proporcionar por lo menos un tema oscuro y uno claro para que el interruptor de modo oscuro de Starlight funcione.
+:::
+
+#### `useStarlightUiThemeColors`
+
+**tipo:** `boolean`  
+**por defecto:** `true` si `themes` no está establecido, de lo contrario `false`
+
+Cuando sea `true`, se utilizan las variables CSS de Starlight para los colores de los elementos de la interfaz de usuario del bloque de código (fondos, botones, sombras, etc.), coincidiendo con el [tema de color del sitio](/es/guides/css-and-tailwind/#temas).
```
Comment (Waxer59) [inline docs/src/content/docs/es/reference/configuration.mdx:407]:
```suggestion
Cuando sea `true`, se utilizan las variables CSS de Starlight para los colores de los elementos de la UI del bloque de código (fondos, botones, sombras, etc.), coincidiendo con el [tema de color del sitio](/es/guides/css-and-tailwind/#temas).
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Comment by dreyfus92** [comment]
Muchas gracias por el review @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by delucis — APPROVED** [review]
Looks great! Thanks @dreyfus92 & @Waxer59 ✨

**Pull request merged**